src/Backend/Modules/Pages/Js/Pages.js

```diff
@@ -125,23 +125,54 @@ jsBackend.pages.extras = {
       var $modal = $(this).closest('.modal')
       var separator = ' › '
       var $pageBlockTitle = $modal.closest('[data-role="page-block-wrapper"]').find('[data-role="page-block-title"]')
+      var $pageBlockPreview = $modal.closest('[data-role="page-block-wrapper"]').find('[data-role="page-block-preview"]')
```

Comment:
ideally `$modal.closest('[data-role="page-block-wrapper"]')` should be a variabele, so it's set once en the variable is used here and a line higher